contracts/governance/Governance.sol

```diff
@@ -265,7 +266,7 @@ contract Governance is Initializable, ReentrancyGuard, GovernanceSettings, Versi
         uint256 votingStartTime = p.votingStartTime();
         uint256 votingMinEndTime = p.votingMinEndTime();
         uint256 votingMaxEndTime = p.votingMaxEndTime();
-        bytes32[] memory options = p.options();
+        bytes[] memory options = p.options();
```

Comment:
I have no idea how were you going to convert bytes32 to bytes, but unless you are able to fit each option into a single byte, following checks will fail:
```
require(options.length <= maxOptions(), "too many options");
```